views/radiometric-target/_form.php

```diff
@@ -0,0 +1,169 @@
+
+    <?php 
+    if ($model->isNewRecord) {
+        echo $form->field($model, 'uri')->textInput([
+                'maxlength' => true, 
+                'readonly' => true, 
+                'id' => 'experimentURI', 
+                'value' => Yii::$app->params['baseURI'],
+                'style' => 'background-color:#C4DAE7;',
+                'data-toggle' => 'tooltip',
+                'title' => 'Automatically generated',
+                'data-placement' => 'left'
+            ]);
+    } else {
+        echo $form->field($model, 'uri')->textInput([
+                'readonly' => true, 
+                'style' => 'background-color:#C4DAE7;',
+            ]);
+    }
+    ?>
+
+    <?= $form->field($model, 'label')->textInput(['maxlength' => true]) ?>
+    
+    <?= $form->field($model, 'brand')->textInput(['maxlength' => true]) ?>
+
+    <?= $form->field($model, 'serialNumber')->textInput(['maxlength' => true]) ?>
+    
+    <?= $form->field($model, 'inServiceDate')->widget(\kartik\date\DatePicker::className(), [
+        'options' => [
+            'placeholder' => Yii::t('app', 'Enter in service date'), 
+            'id' => 'rtInServiceDate'],            
+        'pluginOptions' => [
+            'autoclose' => true,
+            'format' => 'yyyy-mm-dd'
+        ]
+    ]) ?>
+    
+    <?= $form->field($model, 'dateOfPurchase')->widget(\kartik\date\DatePicker::className(), [
+        'options' => [
+            'placeholder' => Yii::t('app', 'Enter date of purchase'), 
+            'id' => 'rtDateOfPurchase'],            
+        'pluginOptions' => [
+            'autoclose' => true,
+            'format' => 'yyyy-mm-dd'
+        ]
+    ]) ?>
+    
+    <?= $form->field($model, 'dateOfLastCalibration')->widget(\kartik\date\DatePicker::className(), [
+        'options' => [
+            'placeholder' => Yii::t('app', 'Enter date of last calibration'), 
+            'id' => 'rtDateOfLastCalibration'],            
+        'pluginOptions' => [
+            'autoclose' => true,
+            'format' => 'yyyy-mm-dd'
+        ]
+    ]) ?>
+    
+    <?= $form->field($model, 'personInCharge')->widget(\kartik\select2\Select2::classname(),[
+            'data' => $this->params['listContacts'],
+            'options' => [
+                'value' => [],
+            ],
+            'pluginOptions' => [
+                'allowClear' => false
+            ],
+        ]);
+    ?>
+    
+    <?= $form->field($model, 'material')->widget(\kartik\select2\Select2::classname(),[
+            'data' => [
+                'carpet' => Yii::t('app', 'Carpet'),
+                'painting' => Yii::t('app', 'Painting'),
+                'spectralon' => Yii::t('app', 'Spectralon')
+            ],
+            'options' => [
+                'value' => [],
+            ],
+            'pluginOptions' => [
+                'allowClear' => false
+            ],
+        ]);
+    ?>
+    
+    <script>
+        function rtChangeFigure(value) {
```

Comment:
you should change the name of this function, maybe remove rt

Reply:
I change it to radiometricTargetChangeFigure